src/midgard/util.cc

```diff
@@ -128,6 +128,20 @@ void trim_shape(float start,
   }
 }
 
+// Generate leg shape in geojson format.
+baldr::json::MapPtr geojson_shape(const std::vector<PointLL> shape) {
```

Comment:
this was in the osrm serializer before. moved it here, though we might actually have it better in `baldr/json.h`?

Reply:
midgard should not have access to baldr, if it does, which this suggests it does then we have lost the plot :smile: 

since this is for serialization why not put it in the tyr serializers header?